README.md

```diff
@@ -76,6 +81,60 @@ All releases can be found [here](https://github.com/okp4/okp4d/releases).
 `okp4d` follows the [Semantic Versioning 2.0.0](https://semver.org/) to determine when and how the version changes, and
 we also apply the philosophical principles of [release early - release often](https://en.wikipedia.org/wiki/Release_early,_release_often).
 
+## Docker image
+
+A docker image is available on [Docker hub](https://hub.docker.com/r/okp4/okp4d):
+
+```bash
+docker pull okp4/okp4d:latest
+```
+
+### Get the documentation
+
+```bash
+docker run okp4/okp4d:latest --help
+```
+
+### Query a running network
+
+Example:
+
+```bash
+API_URL=https://api.devnet.okp4.network:443/rpc
+WALLET=okp41pmkq300lrngpkeprygfrtag0xpgp9z92c7eskm
+
+docker run okp4/okp4d:latest query bank balances $WALLET --chain-id okp4-devnet-1 --node $API_URL
+ ```
+
+### Create a wallet
+
+```bash
+docker run -v $(pwd)/home:/home okp4/okp4d:latest keys add my-wallet --keyring-backend test --home /home 
+```
+
+### Start a node
+
+Everything you need to start a node and even more is explained here : <https://docs.okp4.network/docs/nodes/run-node>
+
+```bash
+MONIKER=node-in-my-name
+CHAIN_ID=localnet-okp4-1
+
+docker run -v $(pwd)/home:/home okp4/okp4d:latest init $MONIKER --chain-id $CHAIN_ID --home /home 
+```
+
+This will create a home folder, you can then update the `config/genesis.json` with one of this ones : <https://github.com/okp4/okp4d/tree/main/chains/>
```

Comment:
```suggestion
This will create a home folder, you can then update the `config/genesis.json` with one of this ones: <https://github.com/okp4/okp4d/tree/main/chains/>
```

Reply:
Hey, I think that's the french way: https://en.wikipedia.org/wiki/Colon_%28punctuation%29#Spacing

Reply:
> Hey, I think that's the french way: https://en.wikipedia.org/wiki/Colon_%28punctuation%29#Spacing

Right! I've personally learned typesetting/typography with Latex 😁

Reply:
:smiley: I don't want any trouble though  :cop: is it alright if I merge as is?

Reply:
The rule is:
> In modern english printing, no space is placed before a colon and a single space is placed after it. Regarding typography, a _thin_ space would be traditionally correct before a colon, but never a _full_ space.

This is what is widely adopted, as far as I can see, and certainly the most correct form. But, well it's up to you to decide.

Reply:
Ok : one day I'll try to find the `thin space` char...